src/agent/dynamic-library/src/windows.rs

```diff
@@ -143,8 +146,8 @@ impl ImageGlobalFlags {
 
     pub fn set_value(&self, value: u32) -> Result<(), ImageGlobalFlagsError> {
         self.create_key()?
-            .set_value(GFLAGS_KEY_NAME, &value)
-            .map_err(|source| ImageGlobalFlagsError::AccessValue {
+            .set_value(GFLAGS_VALUE_NAME, &value)
```

Comment:
Why the naming change ? It looks like it is being used a key here

Reply:
I'm glad I'm not the only one who finds that weird 😅. I'm aligning with the `winreg` API and official registry docs. In registry jargon, a "key" is a path-like object. The object that you and I would call a "key" is really called a "value name", and what we'd want to call the "value" is the "value data".

So to compare registry naming conventions vs. more common naming conventions:
| Registry | Common |
| ------- | -------|
| Key | Path |
| Value Name | Key |
| Value Data | Value |

See the worked example [here](https://docs.microsoft.com/en-us/windows/win32/sysinfo/structure-of-the-registry), as well as this comment:
> Each value consists of a value name and its associated data, if any.